### DISCUSSION TITLE
FOSDEM and FOSDEM junior 2025

### DISCUSSION THREAD

**Discussion by PeterMathijssen** [micropython/micropython#16186] [General]
Is someone from the MicroPython developers or experienced users willing to do a presentation or workshop with children during FOSDEM in Brussels Belgium next year?

There will be 2 rooms for workshops for children on both days and 1 room for presentations on Sunday. 

If you are interested to visit or would be part of a workshop or presentation please read these Call for Participations.

https://lists.fosdem.org/pipermail/fosdem/2024q4/003547.html

https://lists.fosdem.org/pipermail/fosdem/2024q4/003558.html

We would love to have MicroPython there, together with a number of other programming languages. MicroBlocks and Snap! already will be  there doing presentations and workshops.

**Comment by Josverl** [comment]
Hi Peter, I would be interested in that, an d live in the Netherlands (as well ) so travel is not a problem either.

Lets have a chat to discuss

**Comment by PeterMathijssen** [comment]
Hi Jos, that is great. Could you submit a proposal? It could be for both devrooms if you want. You can then tell what you could do, and also maybe tell what you need from us. We could use the FOSDEM system to discuss further.

**Comment by Josverl** [comment]
**Devroom session**
I've drafted one proposal for a [devroom session on micropython-stubs](https://pretalx.fosdem.org/fosdem-2025/me/submissions/TBRYKH/) 

I welcome suggestions and opinions for improvement 

There are a few different devrooms that this might fit in ( [Python](https://lists.fosdem.org/pipermail/fosdem/2024q4/003564.html) , [Embedded](https://lists.fosdem.org/pipermail/fosdem/2024q4/003575.html), perhaps [Tool the Docs](https://lists.fosdem.org/pipermail/fosdem/2024q4/003571.html)) 
 
 Another question is delivering a workshop for a workshop for ~30 children.
 I have not decided yet , and would welcome help on that.
 
 It would be cool to teach them to build something combining 
 - Physical + Code + visually appealing +  ( a bit of AI ?) that they can also take home with them.

I was thinking off creating a simple neopixel lightsaber using a small 
RP2040-Zero board and a movement sensor to trigger some effects, but am concerned on the balance/entanglement  between hardware and software, and want to avoid soldering at all costs
 If you have suggestions for a simple build-project or boards / components that could be used please reach out.  
 
 - [ ] RP2040-Zero
 - [ ] [Cytron maker-nano-rp2040](https://www.cytron.io/p-maker-nano-rp2040-simplifying-projects-with-raspberry-pi-rp2040)

**Comment by jonnor** [comment]
Great initiative! I might be at the next FOSDEM. However, I already have 3 proposals submitted for various devrooms (Embedded, Python and Low level AI). So I do not have capacity to submit for the educational one :o
But, I would be happy to meet up with others developing and using MicroPython :)

**Comment by jonnor** [comment]
Hi. Is anyone else going to FOSDEM 2025? I will be there and present MicroPython in the Embedded devroom. And also give a talk in the "low-level AI hacking" devroom. Would be cool to meet up with other MicroPython users, contributors, or just those that are curious about it :)

**Reply by andypiper** [reply]
I’ll be there mostly on the Mastodon table and at the social web devroom, but always want to meet MicroPython folks as well!

**Reply by Josverl** [reply]
I might pop in for a day, not sure yet which dat. And yes, it would like to meet up

**Comment by jonnor** [comment]
Two MicroPython related from FOSDEM talks now up on:

MicroPython - Python for microcontrollers and embedded linux (FOSDEM 2025)
https://www.youtube.com/watch?v=8Ao7DsTkpS4

Milliwatt-sized Machine Learning on Microcontrollers with emlearn (FOSDEM 2025)
https://www.youtube.com/watch?v=L534ngXv8I8
